arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1110 @@
+          "type": "string",
+          "description": "The search result type."
+        },
+        "total": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The total search results."
+        },
+        "top": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The number of top search results."
+        },
+        "id": {
+          "type": "string",
+          "description": "The id of the search results request."
+        },
+        "CoreSummaries": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CoreSummary"
+          },
+          "description": "The core summaries."
+        },
+        "Status": {
+          "type": "string",
+          "description": "The status of the search results."
+        },
+        "StartTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The start time for the search."
+        },
+        "LastUpdated": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time of last update."
+        },
+        "ETag": {
+          "type": "string",
+          "description": "The ETag of the search results."
+        },
+        "sort": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SearchSort"
+          },
+          "description": "Gets or sets sort."
+        },
+        "requestTime": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The request time."
+        },
+        "aggregatedValueField": {
+          "type": "string",
+          "description": "The aggregated value field."
+        },
+        "aggregatedGroupingFields": {
+          "type": "string",
+          "description": "The aggregated grouping fields."
+        },
+        "sum": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The sum."
+        },
+        "max": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The max."
+        },
+        "schema": {
+          "$ref": "#/definitions/SearchMetadataSchema",
+          "description": "The schema."
+        }
+      },
+      "description": "Metadata for search results."
+    },
+    "SavedSearch": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The id of the saved search."
+        },
+        "etag": {
+          "type": "string",
+          "description": "The etag of the saved search."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SavedSearchProperties",
+          "description": "Gets or sets properties of the saved search."
+        }
+      },
+      "description": "Value object for saved search results."
+    },
+    "SavedSearchesListResult": {
+      "properties": {
+        "__metadata": {
```

Comment:
This API was implemented long time ago so it is not using the best practices. I double checked and can confirm that it does not support nextLink and is using __metadata. Since customers are already using this API like this, I will keep the swagger matching the implementation.